INFO     [2022-01-25 11:16:46] --- Reachable set interface initialized. (reach_interface.py:23)
INFO     [2022-01-25 11:16:46] --- * Computing reachable sets... (reach_interface.py:88)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 1 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 2 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 3 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 4 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 5 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 6 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 7 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 8 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 9 (reach_set_py.py:54)
DEBUG    [2022-01-25 11:16:46] --- Computing reachable set for time step 10 (reach_set_py.py:54)
INFO     [2022-01-25 11:16:46] --- Pruning nodes not reaching final time step. (reach_set_py.py:86)
INFO     [2022-01-25 11:16:46] --- 	#Nodes before pruning: 	11 (reach_set_py.py:115)
INFO     [2022-01-25 11:16:46] --- 	#Nodes after pruning: 	11 (reach_set_py.py:119)
INFO     [2022-01-25 11:16:46] --- 	Computation took: 	0.154s (reach_interface.py:95)
INFO     [2022-01-25 11:16:50] --- Done. (compute_reachable_set.py:26)
